Client/Frontend/Home/RemoteTabsPanel.swift

```diff
@@ -35,58 +34,81 @@ private struct RemoteTabsPanelUX {
     static let EmptyStateInstructionsSyncTabsPasswordsString = NSLocalizedString("Sync your tabs, passwords and more.", comment: "Sync tabs and passwords empty state instructions.")
 
     static let EmptyStateInstructionsGetTabsBookmarksPasswordsString = NSLocalizedString("Get your open tabs, bookmarks, and passwords from your other devices.", comment: "A re-worded offer about Sync that emphasizes one-way data transfer, not syncing.")
+
+    static let HistoryTableViewHeaderChevronInset: CGFloat = 10
+    static let HistoryTableViewHeaderChevronSize: CGFloat = 20
+    static let HistoryTableViewHeaderChevronLineWidth: CGFloat = 3.0
 }
 
 private let RemoteClientIdentifier = "RemoteClient"
 private let RemoteTabIdentifier = "RemoteTab"
 
-class RemoteTabsPanel: UITableViewController, HomePanel {
+class RemoteTabsPanel: UIViewController, HomePanel {
     weak var homePanelDelegate: HomePanelDelegate? = nil
+    private lazy var tableViewController = {
```

Comment:
Nit: Don't think you need the closure here. You can just do `= UITableViewController()`